clients/web/src/styles.css

```diff
@@ -6,8 +6,8 @@
 @layer base {
   /** fonts */
   :root {
-    --header-height: 69px;
-    --toolbar-height: 28px;
+    --header-height: 90px;
```

Comment:
This seems to create a gap between the border and the following tab content:

![chrome_ZHlR3BAKhm](https://github.com/crabnebula-dev/devtools/assets/119593300/d614943a-da74-4114-bcaa-f086f29af371)